lib/giraph/remote/mutation.rb

```diff
@@ -0,0 +1,12 @@
+module Giraph
+  module Remote
+    # Field resolver to plug a remote GraphQL mutation root into a local type
+    class Mutation < Query
```

Comment:
Considering how this is used:

```rb
field :remote do
  type RemoteService::Types::QueryRoot
  description 'Remote Service GraphQL entry point'
  resolve Giraph::Remote::Mutation.new('http://localhost:3001/graph')
  # resolve Giraph::Remote::Query.new('http://localhost:3001/graph') # for Query
end
```

I'd like to keep `Mutation` around for better readability. 

However, I agree that `@mutation` is extra. Maybe I can have a factory method on class like `connect` which takes the same arguments as `new` but passes in the `Connector` class with `mutation` flag set. That should still help with testing, and keep `mutation` out of instance. What do you think?